babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    presets: [
+        [
+            '@babel/preset-env',
+            {
+                targets: '> 1%, last 2 versions, Firefox ESR',
```

Comment:
Let’s make sure this matches the setting we have defined in cornerstone

Reply:
@junedkazi yes, we grabbed it from Cornerstone
https://github.com/bigcommerce/cornerstone/blob/master/webpack.common.js#L33